src/types/LintConfig.ts

```diff
@@ -67,9 +68,13 @@ export class LintConfig {
       this.lineLintRules.pop()
     }
 
-    this.lineLintRules.push(maxLineLength)
-    if (!isNaN(json?.maxLineLength)) {
+    if (json?.maxLineLength > 0) {
```

Comment:
I think that it makes sense to check both that `maxLineLength` is not `NaN` and is greater than `0`

Reply:
I think there's no need to check if `maxLineLength` is not `NaN` because if `maxLineLength` is NaN then condition `json?.maxLineLength > 0` will always be false. I could not think of any scenario where it will be true.